src/tetris.js

```diff
@@ -15,30 +15,30 @@
 M=e=>P&&(
     h=0,
     p=P.map((v,i)=>(
+        E=~i%2,
         x=eval(e[0]),
-        i%2
-            ?b[h|=x<0|x>9|b[y]>>x&1|y>17,y]|=1<<x
-            :0,
+        E
+            ?0
+            :b[h|=x<0|x>9|b[y]>>x&1|y>17,y]|=1<<x,
         y=x
     ),b=[...B]),
     h
 ),
 b=C=Array(9),
-S=P=k=0,
 D=e=>S|=b.map(v=>{S+=`
 `;for(x=10;x--;)Z.innerText=S+='□■'[v^1023+P&&v>>x&1]},P=p),
-(_=e=>D(
+(_=s=>D(
     P=P||[...'021213030405061617'.substr(
         new Date%6*2,
         8,
-        s=0,
         B=[...C,...b].filter(v=>v^1023||!(S+=++s)).slice(-18)
     )],
-    M`v-~i%2`&&M`v`&(p=k=0),
-    setTimeout(_,k-2&&200)
+    setTimeout(_,k-2&&200,M`v-E`&&M`v`&(p=k=0))
```

Comment:
Unfortunately, you can’t combine the statements like that. `k` has to be set before `k-2` is evaluated or the pause to show the removed lines will not happen for accelerated pieces.